helpers/convertCurrency_helper.js

```diff
@@ -0,0 +1,13 @@
+const Helper = require('@codeceptjs/helper');
+
+class ConvertCurrency extends Helper {
+  async convertUsdToUah() {
```

Comment:
According to method's name, it should convert, but it only returns current rate. Add input parameter - price in USD and change method to do conversion.

Reply:
Changed.